### ISSUE TITLE
Schema tableName config issue.

### ISSUE THREAD

**Issue by pavanmishra** [ryanfitz/vogels#37]
```
var vogels = require('vogels');

vogels.AWS.config.loadFromPath('config/credentials.json');
var Account = vogels.define('Account', function (schema) {
  schema.String('email', {hashKey: true});
  schema.String('name').required(); // name attribute is required
  schema.Number('age'); // age is optional
  schema.Date('created', {default: Date.now});
});
Account.config({tableName: 'Account'});
Account.create({email: 'foo@example.com', name: 'Foo Bar', age: 21}, function (err, acc) {
  console.trace(err);
});
```

fails with 

```
Trace: { [ResourceNotFoundException: Requested resource not found]
  message: 'Requested resource not found',
  code: 'ResourceNotFoundException',
  time: Tue Aug 12 2014 23:18:58 GMT+0530 (IST),
  statusCode: 400,
  retryable: false }
```

Issue goes away on removing

`Account.config({tableName: 'Account'});`

**Comment by deepaksisodiya** [comment]
hey, same issue i was facing,
creating table manually works for me

vogels.createTables({
  'Account'  : {readCapacity: 1, writeCapacity: 10},
}, function (err) {
  if(err) {
    console.log('Error creating tables', err);
    process.exit(1);
  }
  console.log('created');
});

I hope, this will help :)

**Comment by talvor** [comment]
What is the reasoning behind lowercase, pluralize table names?

**Comment by pauldps** [comment]
I recently came across this problem and here are my findings:

If you set `tableName` in your schema, setting them using `Model.config({tableName: value})` will not override the previous value (which may lead to ResourceNotFound errors). I had to delete `tableName` from my schema to get the `Model.config(...)` approach to work.

In my opinion, setting `tableName` via `Model.config(...)` should overwrite whatever value was defined previously, via schema or not.
